dockerfiles/cli/scripts/cmd_config.sh

```diff
@@ -95,6 +94,17 @@ generate_configuration_with_puppet() {
     fi
   fi
 
+  for element in "${CLI_ENV_ARRAY[@]}" 
+  do
+    var1=$(echo $element | cut -f1 -d=)
+    var2=$(echo $element | cut -f2 -d=)
+
+    if [[ $var1 == CHE_* ]] ||
+       [[ $var1 == ${CHE_PRODUCT_NAME}_* ]]; then
```

Comment:
It does not.  In the case of ARTIK it would be ARTIK_*.  And in the case of Codenvy and Codenvy SAAS the would both be CODENVY_*.